### ISSUE TITLE
Default privileges lack role for which are created

### ISSUE THREAD

**Issue by michal-bartak** [dbeaver/dbeaver#23419]
### Description

After issuing following statements:

```sql
ALTER DEFAULT PRIVILEGES FOR ROLE sazky IN SCHEMA clients GRANT SELECT ON TABLES TO app_client;
ALTER DEFAULT PRIVILEGES FOR ROLE sazky IN SCHEMA clients GRANT SELECT ON TABLES TO public;
```
The Source pane of a schema shows:
```sql
ALTER DEFAULT PRIVILEGES IN SCHEMA clients GRANT SELECT ON TABLES TO public;
ALTER DEFAULT PRIVILEGES IN SCHEMA clients GRANT SELECT ON TABLES TO app_client;
```

See the missing `FOR ROLE sazky` part.

Postgresql doesn't provide the option to create a default for everyone. The default is always created for a specific role logged in.  The syntax however provides a shortcut to use the currently logged role. Example:

ALTER DEFAULT PRIVILEGES FOR ROLE michal IN SCHEMA clients GRANT SELECT ON TABLES TO app_client;
is equal to connecting to the database as michal and then issuing
ALTER DEFAULT PRIVILEGES IN SCHEMA clients GRANT SELECT ON TABLES TO app_client;
That's all.

Thus SQL shown by DBeaver is incomplete/bugged

### DBeaver Version

CE 24.0.2

### Operating System

Fedora 39

### Database and driver

Postgresql

### Steps to reproduce

See description

**Comment by E1izabeth** [comment]
Thank you for bug report

**Comment by michal-bartak** [comment]
Hello, just a reminder. 
Such generated DDL is dangerous!

**Comment by E1izabeth** [comment]
Yes. I agree. Thank you for drawing our attention to it.

**Comment by labanovichttps** [comment]
@michalk-k Hello, we could't reproduce this bug. Could you try it on the latest DBeaver version?

**Comment by michal-bartak** [comment]
Yeah, it has been fixed already in some previous versions of DBeaver. I've seen it in a changelog.

Closing this issue then
